peps/pep-0764.rst

```diff
@@ -214,14 +224,18 @@ Mypy supports a similar syntax as an :option:`experimental feature <mypy:mypy.--
     def test_values() -> {"int": int, "str": str}:
         return {"int": 42, "str": "test"}
 
+Support for this PEP is added in `this pull request <https://github.com/python/mypy/pull/18889>`_.
```

Comment:
To fix the build, add a second underscore to make an "anonymous" reference, so it's not duplicated:
```suggestion
Support for this PEP is added in `this pull request <https://github.com/python/mypy/pull/18889>`__.
```
Plus we could use more accessible link text instead:
https://www.a11yproject.com/posts/creating-valid-and-accessible-links/#write-helpful-link-text

Reply:
Do you have suggestions to make it more accessible? The way I understand what's described in your link is that the link text should describe what it points to, and _this pull request_ seems to describe the target correctly? (Perhaps mypy should be mentioned in the text).

Reply:
If someone is navigating just via links, they'll hear "this pull request", "this pull request" for both.

Perhaps we could mention the repo names? For example, "this mypy pull request", "this typing_extensions pull request".